Frontend/android-native/app/src/main/java/in/juspay/mobility/CommonJsInterface.java

```diff
@@ -3419,7 +3419,7 @@ public void removeMediaPlayer() {
     }
 
     @JavascriptInterface
-    public void updateRoute(String json, String dest, String eta, String specialLocation) {
+    public void updateRoute(String json, String src, String dest, String eta, String specialLocation) {
```

Comment:
Handle backward compatibility

Reply:
Updated the JBridge.js file to handle backward compatibility